native_app/pubspec.yaml

```diff
@@ -0,0 +1,28 @@
+name: native_app
+description: >
+  A command line application that can be compiled to native code using
+  dart2native.
+
+environment:
+  sdk: '>=2.5.0 <3.0.0'
+
+dependencies:
+  args: ^1.0.0
+  build_cli_annotations: ^1.0.0
```

Comment:
Is this still needed?

Reply:
Yeah this is still needed until we fix #17